frontend/src/upload.js

```diff
@@ -181,19 +210,34 @@ $(document).ready(function() {
     const link = document.createElement('td');
     const expiry = document.createElement('td');
     const del = document.createElement('td');
-    del.setAttribute('align', 'center');
-    const btn = document.createElement('button');
     const popupDiv = document.createElement('div');
-    const $popupText = $('<span>', { class: 'popuptext' });
+    const $popupText = $('<div>', { class: 'popuptext' });
     const cellText = document.createTextNode(file.name);
 
+    const url = file.url.trim() + `#${file.secretKey}`.trim();
+    $('#link').attr('value', url);
+    $('#copy-text').html(
+      'Copy and share the link to send your file: ' + file.name
+    );
+    $popupText.attr('tabindex', '-1');
+
     name.appendChild(cellText);
 
     // create delete button
-    btn.innerHTML = 'x';
-    btn.classList.add('delete-btn');
-
-    link.innerHTML = file.url.trim() + `#${file.secretKey}`.trim();
+    del.innerHTML =
+      '<span class=\'icon-cancel-1\' title=\'Delete\' style=\'margin-left: -7px\'></span>';
+
+    link.innerHTML = '<span class=\'icon-docs\' title=\'Copy URL\'></span>';
```

Comment:
Can we get some feedback on clicking this just to show that it has been copied, a slight darkening of the colour, or bolding the font, at a minimum.